Nim.cs

```diff
@@ -11,9 +11,16 @@ enum Player
             Computer
         }
 
+        enum Variant
+        {
+            Nim21,
+            Nim100
+        }
+
         int Total { get; set; }
         Player CurrentPlayer { get; set; }
         bool GameOver { get; set; }
+        Variant gameType { get; set; }
```

Comment:
Let's rename this to `GameType` so that it's clear that it's a property rather than a local variable.